grammars.yml

```diff
@@ -58,6 +58,7 @@ vendor/grammars/NimLime:
 - source.nim_filter
 - source.nimcfg
 vendor/grammars/Nixinova-tmLanguage:
+- text._redirects
```

Comment:
**Nitpick:** This should be `source.`, not `text.`. The two scope-bases roughly correspond to the following language types:

`source.` ⇒ `data`, `programming`  
`text.` ⇒ `markup`, `prose`

I'd also rename `_redirects` to `redirects`. A good scope-name should match:

~~~regexp
/^(source|text)\.[a-z]+(-[a-z]+)*$/
~~~

So altogether:

```suggestion
- source.redirects
```